labellines/test.py

```diff
@@ -203,3 +203,15 @@ def test_negative_spacing():
 
     # Should not throw an error
     labelLine(line, 0.2, label='Test')
+    
+def test_yoffset():
+    x = np.linspace(0, 1)
+
+    plt.plot(x, np.sin(x), label=r'$\sin x$')
+    plt.plot(x, np.cos(x), label=r'$\cos x$')
+
+    labelLines(plt.gca().get_lines(), xvals=(0, .7), align=False, yoffsets= [-.1, .08], bbox={'alpha': 0})
+    return plt.gcf()
+
+test_yoffset()
+
```

Comment:
The tests are automatically triggered, so there's no need to call them manually. Also, I'd

```suggestion
def test_yoffset():
    x = np.linspace(0, 1)

    for yoffset in (
        [-.1, .08],   # try lists
        1,            # try int
        2.0           # try float
    ):
        plt.clf()
        ax = plt.gca()
        ax.plot(x, np.sin(x), label=r'$\sin x$')
        ax.plot(x, np.cos(x), label=r'$\cos x$')
        lines = ax.get_lines()
        labelLines(lines, xvals=(0, .7), align=False, yoffsets=yoffset, bbox={'alpha': 0})
```

Reply:
Can I suggest the following test? Otherwise the offsets are off the chart. I multiplied the sin and cos by 10 and changed the offsets a bit

''''
def test_yoffset():
    x = np.linspace(0, 1)

    for yoffset in (
        [-0.5, 0.5],   # try lists
        1,            # try int
        1.2           # try float
    ):
        plt.clf()
        ax = plt.gca()
        ax.plot(x, np.sin(x)*10, label=r'$\sin x$')
        ax.plot(x, np.cos(x)*10, label=r'$\cos x$')
        lines = ax.get_lines()
        labelLines(lines, xvals=(0.2, 0.7), align=False, yoffsets=yoffset, bbox={'alpha': 0})
'''